news/2018-12-18-osu-world-cup-2018-recap.md

```diff
@@ -0,0 +1,53 @@
+
+![](/wiki/shared/news/2018-12-18-osu!-world-cup-2018-recap/us-uk.jpg) 
+
+(A mere 70 points difference to give the American players a PTSD attack)
+
+The Tiebreaker map – [The Azure Arbitrator](https://osu.ppy.sh/beatmapsets/427166#osu/922172) – was a gruesome 8-minute test of high speed consistency, and widely regarded as a deserving but also hard map choice for its purpose. Going into the map, it came as no surprise when the American team wavered and collapsed combos. However, what was surprising was that when the crowd discovered Bubbleman holding combo, maintaining that incredible score growth for Team UK all the way up to more than 2467 before breaking! This amazing feat alone accounted for more than 1/3 of Team USA's score at the end, giving Team UK the much-needed bracket reset, and paved way for the second match!
+
+### [United States vs United Kingdom game 2](https://osu.ppy.sh/community/matches/48149847)
+
+After a quick break the teams returned for the final showdown of the year. At this point, Twitch channel viewer count broke 13,000, and hype was in the air. Tournament organizer HappyStick went over and congratulated Bubbleman in person, who only responded: "Not yet. It’s not done yet. I'm not done yet."
+
+After witnessing what Team UK could do on [Unravel](https://osu.ppy.sh/beatmapsets/731933#osu/1544460), Team USA adjusted strategy and banned that map instead. The two teams traded blows until arriving at 2-2, when Team USA saw an opening and went in for the offense, taking 2 more points, bringing the score to 4-2. Team UK quickly reorganized a counterattack, tying the score at 4-4. However, fatigue is definitely showing among the players, and the almost 3-hour long battle is taking its toll! Team UK soon discovered their consistency waning on [Bassdrop Freaks](https://osu.ppy.sh/beatmapsets/806859#osu/1693575). Team USA followed up with their signature pick Tabi to Ferry. Finally, Bad Elixir sealed the fate for Team UK, giving the win to Team USA at 7-4!
+
+### Results
+
+![](/wiki/shared/news/2018-12-18-osu!-world-cup-2018-recap/podium.png)
```

Comment:
There is quite a bit of whitespace in the top portion of the podium image; otherwise, everything else looks good to me.